### PR TITLE
Apply the new doc structure

### PR DESCRIPTION
**Pull request by alexzhc**
This pr is meant to apply the new doc structure per issue https://github.com/hwameistor/hwameistor/issues/132, with previously existing materials, both in English and Chinese.

### CODE REVIEW

File: docs/i18n/cn/docusaurus-plugin-content-docs/current/1.whats_the_hwameistor.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: "intro"
 sidebar_position: 1
-sidebar_label: "什么是 HwameiStor"
+sidebar_label: "1. 什么是 HwameiStor"
```
Comment (windsonsea) [inline docs/i18n/cn/docusaurus-plugin-content-docs/current/1.whats_the_hwameistor.md:4]:
```suggestion
sidebar_label: "什么是 HwameiStor"
```

File: docs/docs/1.whats_the_hwameistor.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 id: "intro"
 sidebar_position: 1
-sidebar_label: "What is HwameiStor"
+sidebar_label: "1. What is HwameiStor"
```
Comment (windsonsea) [inline docs/docs/1.whats_the_hwameistor.md:4]:
```suggestion
sidebar_label: "What is HwameiStor"
```

**Review by windsonsea — COMMENTED** [review]
wow, thanks for your great efforts involved with so many files. Some suggestions:
1. For a docsite hosted on web, we need not to add 1.1, 2.1, 3.1 to the title, which is acceptable for a filename to easily recognize by yourself.
2. Open a PR for each language. Too many files and multi-language may cause much difficulty to review.
3. Do not changed too many files in a PR, which may spend much time to review and merge.

**Comment by alexzhc** [comment]
@sun7927 what is your opinion about removing index in catalogs per @windsonsea's suggestion

**Comment by alexzhc** [comment]
@windsonsea 

> 1. For a docsite hosted on web, we need not to add 1.1, 2.1, 3.1 to the title, which is acceptable for a filename to easily recognize by yourself.

good point, removed.

> 2. Open a PR for each language. Too many files and multi-language may cause much difficulty to review.
When file names mismatch, docusaurus will default translation to primary one which is english. That's why both language versions are updated the same time here.

> 3. Do not change too many files in a PR, which may spend much time to review and merge.

It is a complete overhaul. Further updates will be smaller PR.

File: docs/docs/5.applications/5.1.overview.md [hunk 3]
```diff
@@ -19,13 +19,13 @@ For the non-HA local LVM volume, it's the best solution for data persistency in
 For the HA local LVM volume, it's the best solution for data persistency in the following use cases:
 
 - **Database**, such as MySQL, PostgreSQL, etc.
-- Other applications which require the data with HA features.
+- Other applications which requires the data with HA features.
```
Comment (windsonsea) [inline docs/docs/5.applications/5.1.overview.md:22]:
```suggestion
- Other applications which require the data with HA features.
```

**Review by windsonsea — APPROVED** [review]

**Pull request merged**